storyscript/ErrorCodes.py

```diff
@@ -6,15 +6,15 @@ class ErrorCodes:
     unidentified_error = ('E0001', '')
     service_name = ('E0002', "A service name can't contain `.`")
     arguments_noservice = ('E0003',
-                           'You have defined an argument, but not a service')
+                           'Unefined argument; not a service')
```

Comment:
```suggestion
                           'Undefined argument; not a service')
```

Semi-colon shouldn't be used in error messages as they might confuse tools.